DependencyInjection/DoctrinePHPCRExtension.php

```diff
@@ -68,6 +68,7 @@ public function load(array $configs, ContainerBuilder $container)
         $this->loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
 
         $this->loader->load('phpcr.xml');
+        $this->loader->load('commands.xml');
```

Comment:
just for my understanding: did we previously rely on autodiscovery? and is that not good in symfony 4 anymore?

Reply:
This does not work anymore in 4.0. And should have thrown a deprecation warning in 3.4, but didn't.